### PR TITLE
Update/audit code owners

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Cleaning code owners

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -20,14 +20,11 @@ src/site/includes @department-of-veterans-affairs/vsa-public-websites-frontend @
 src/site/components @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-caregiver-frontend
 
 # Public Websites
-src/platform/site-wide @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/frontend-review-group
```
Comment (timwright12) [inline .github/CODEOWNERS:23]: nothing in here

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -20,14 +20,11 @@ src/site/includes @department-of-veterans-affairs/vsa-public-websites-frontend @
...
 src/site/includes/debug.drupal.liquid @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/vsa-facilities-frontend
 src/site/layouts/campaign_landing_page.drupal.liquid @department-of-veterans-affairs/vsa-public-websites-frontend
 
 # Facility Locator and VAMC pages
 src/site/layouts @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
-src/platform/site-wide/sass/modules/_m-side-nav.scss @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
-src/platform/site-wide/side-nav @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
```
Comment (timwright12) [inline .github/CODEOWNERS:30]: these files don't exist anymore

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -44,7 +41,4 @@ src/site/paragraphs/facilities @department-of-veterans-affairs/vsa-facilities-fr
 src/site/stages/build/drupal @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/vsa-facilities-frontend
 
 # Liquid template filters
-src/site/filters @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
-
-# Metalsmith plugins
-src/site/stages/build/plugins/modify-dom @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
```
Comment (timwright12) [inline .github/CODEOWNERS:50]: owned by platform

**Review by ndsprinkle — APPROVED** [review]

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -9,42 +9,23 @@
...
-src/platform/site-wide/sass/modules/_m-side-nav.scss @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
-src/platform/site-wide/side-nav @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
 src/site/facilities @department-of-veterans-affairs/vsa-facilities-frontend
 src/site/layouts/health*.drupal.liquid @department-of-veterans-affairs/vsa-facilities-frontend
 src/site/navigation/facility_no_drupal_page_sidebar_nav.drupal.liquid @department-of-veterans-affairs/vsa-facilities-frontend
 src/site/navigation/facility_sidebar_nav.drupal.liquid @department-of-veterans-affairs/vsa-facilities-frontend
-src/site/includes/facilityListing.drupal.liquid @department-of-veterans-affairs/vsa-facilities-frontend
-src/site/includes/vet_centers @department-of-veterans-affairs/vsa-facilities-frontend
-src/site/components/phone.drupal.liquid @department-of-veterans-affairs/vsa-facilities-frontend
-src/site/components/phone-number.drupal.liquid @department-of-veterans-affairs/vsa-facilities-frontend
```
Comment (timwright12) [inline .github/CODEOWNERS:38]: redundant rules

**Pull request merged**